functions/Remove-SqlBackupFromDisk.ps1

```diff
@@ -0,0 +1,223 @@
+
+## is the retention value and must be an integer value
+U signifies the units where the valid units are:
+h = hours
+d = days
+w = weeks
+m = months
+
+Formatting Examples: 
+'48h' = 48 hours
+'7d' = 7 days
+'4w' = 4 weeks
+'1m' = 1 month
+
+.PARAMETER CheckArchiveBit
+Check the archive bit on files before deletion
+
+.PARAMETER RemoveEmptyBackupFolders
+Remove any empty folders after the cleanup process is complete.
+
+.NOTES
+Original Author: Chris Sommer, @cjsommer, www.cjsommer.com
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Remove-SqlBackupFromDisk
+
+.EXAMPLE
+Remove-SqlBackupFromDisk -BackupFolder 'C:\MSSQL\Backup\' -BackupFileExtenstion 'trn' -RetentionPeriod '48h'
+
+The cmdlet will remove '*.trn' files from 'C:\MSSQL\Backup\' and all subdirectories that are more than 48 hours. 
+
+.EXAMPLE
+Remove-SqlBackupFromDisk -BackupFolder 'C:\MSSQL\Backup\' -BackupFileExtenstion 'trn' -RetentionPeriod '48h' -WhatIf
+ 
+Same as example #1, but using the WhatIf parameter. The WhatIf parameter will allow the cmdlet show you what it will do, without actually doing it.
+In this case, no trn files will be deleted. Instead, the cmdlet will output what it will do when it runs. This is a good preventatitive measure
+especially when you are first configuring the cmdlet calls. 
+
+.EXAMPLE
+Remove-SqlBackupFromDisk -BackupFolder 'C:\MSSQL\Backup\' -BackupFileExtenstion 'bak' -RetentionPeriod '7d' -CheckArchiveBit
+
+The cmdlet will remove '*.bak' files from 'C:\MSSQL\Backup\' and all subdirectories that are more than 7 days old. 
+It will also ensure that the bak files have been archived using the archive bit before removing them.
+
+.EXAMPLE
+Remove-SqlBackupFromDisk -BackupFolder 'C:\MSSQL\Backup\' -BackupFileExtenstion 'bak' -RetentionPeriod '1w' -RemoveEmptyBackupFolders
+
+The cmdlet will remove '*.bak' files from 'C:\MSSQL\Backup\' and all subdirectories that are more than 1 week old. 
+It will also remove any backup folders that no longer contain backup files.
+
+
+
+#>
+	[CmdletBinding(SupportsShouldProcess=$true)]
+	Param (
+		[parameter(Mandatory = $true)]
+        [ValidateScript({Test-Path $_ -PathType 'Container'})]
+		[string]$BackupFolder ,
+
+		[parameter(Mandatory = $true)]
+		[string]$BackupFileExtenstion ,
```

Comment:
I think that would be a safe option. Either that or once again make them choose which makes them think about it.